package.json

```diff
@@ -60,8 +60,8 @@
     "ignore-by-default": "^1.0.1",
     "minimatch": "^3.1.2",
     "pstree.remy": "^1.1.8",
-    "semver": "^5.7.1",
-    "simple-update-notifier": "^1.0.7",
+    "semver": "^7.5.3",
+    "simple-update-notifier": "^1.1.0",
```

Comment:
Bump this to 2.0.0 to get rid of semver problem.

Reply:
Will do that soon

Reply:
@JimmyBjorklund, shouldn't the `simple-update-notifier` version be `^2.0.0`?

Reply:
You are right it should